channels/us.m3u

```diff
@@ -403,6 +403,8 @@ http://api.new.livestream.com/accounts/10891792/events/3578864/live.m3u8
 https://1329918864.rsc.cdn77.org/xPdw4M5ZQn8H4H2sdDXokQ==,1586798443/ls-54548-1/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="" group-title="News",FOX News
 https://fncgohls-i.akamaihd.net/hls/live/263399/FNCGOHLSv2/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="" group-title="News",FOX News (1)
+http://fnctve-i.akamaihd.net/hls/live/200669/fnctve/HD.m3u8
```

Comment:
Link does not work.

Reply:
Just tried it and works in GSE. Link is from fluxus.

Reply:
Yes you are right the link really works in GSE player, but unfortunately it doesn't work in all other players I know. 

For example, this is what VLC player, Elmedia Video Player and streamtest.in produces:
<img width="497" alt="Captura de pantalla 2020-04-14 a las 20 55 31" src="https://user-images.githubusercontent.com/7253922/79258380-b6c6de00-7e93-11ea-96ba-f03fcac952bf.png">
<img width="319" alt="Captura de pantalla 2020-04-14 a las 20 55 46" src="https://user-images.githubusercontent.com/7253922/79258393-b9c1ce80-7e93-11ea-94f5-390e252c4566.png">
<img width="933" alt="Captura de pantalla 2020-04-14 a las 20 56 03" src="https://user-images.githubusercontent.com/7253922/79258396-ba5a6500-7e93-11ea-8401-4c432683512a.png">

So, just in case, I left a small label with the channel name for the other users.